deploy/src/Microsoft.Azure.IIoT.Deployment/appsettings.json

```diff
@@ -112,7 +112,7 @@
     // azure-industrial-iot Helm chart version
     "ChartVersion": "0.3.2",
     // Azure IIoT components image tag
-    "ImageTag": "2.7.206",
```

Comment:
Please revert this back to `2.7.206`. Updating IAI for `2.8.1` release entails not only change of image tag and chart version but also cleanup of resources that were deployed for microservices that are removed from the deployment. And we have a separate PBI for doing all of that.

Reply:
reverted